_config.yml

```diff
@@ -4,7 +4,7 @@
 domain: machinetranslate.org
 url: https://machinetranslate.org
 title: Machine Translate
-tagline: Open information and community for machine translation
+tagline: The Machine Translate Foundation
```

Comment:
I think it still needs an actual tagline.

Reply:
Just from an SEO perspective...

Reply:
It'll show up as:
> Machine Translate | The Machine Translate ...
which is not a very good use of real estate.

This is what is currently shown:
<img width="787" alt="Screenshot 2023-10-27 at 00 56 23" src="https://github.com/machinetranslate/machinetranslate.org/assets/11457984/a2124603-bc36-4e49-8616-935b3e0f9d76">

3 options that occur to me:
1. Add "... Foundation" to the title, keep the tagline as it was
2. Add "Foundation for... " to the tagline, keep the title as it was
3. Keep both as they were

I lean towards 1.

Reply:
Yeah, that's true, I agree. I've fixed it.